libs/agno/agno/tools/function.py

```diff
@@ -422,7 +425,42 @@ def _wrap_callable(func: Callable) -> Callable:
             return wrapped
 
     def process_schema_for_strict(self):
```

Comment:
Is this not used elsewhere? Just wondering if there is another flow we need to test